innovation_sweet_spots/getters/nihr.py

```diff
@@ -0,0 +1,25 @@
+"""
+innovation_sweet_spots.getters.nihr
+
+Module for easy access to downloaded NIHR data
+"""
+# import numpy as np 
+# import pandas as pd 
+# import altair as alt
+import os
+from innovation_sweet_spots.utils.read_from_s3 import load_csv_from_s3
+
+def get_nihr_summary_data():
+    """Load NIHR .csv file from S3"""
+    return load_csv_from_s3("inputs/data/nihr/", "nihr_summary_data")
+
+def save_nihr_to_local():
+    """Save NIHR summary data locally"""
+    data = get_nihr_summary_data()
+    path = os.path.abspath("data/") + "/nihr_summary_data.csv"
+    print(path)
+    data.to_csv(path, sep=",")
```

Comment:
I'd suggest to not output index, as otherwise it will make the table a bit more messy when loading it in
```suggestion
    data.to_csv(path, sep=",", index=False)
```